### ISSUE TITLE
[BUG] Holograms won't go away after removing shop

### ISSUE THREAD

**Issue by haloflooder** [Ghost-chu/QuickShop-Reremake#770]
**Describe the bug**
When you remove a shop, the item/hologram comes back after a second or two

**To Reproduce**
1. Create a random shop
2. Remove the shop
3. The hologram comes back after a second or two

**Expected behavior**
The hologram shouldn't come back because the shop doesn't exist anymore.

**Screenshots**
![Image](https://i.imgur.com/u4hXcrz.png)

**Paste link:**
It failed to create a link (api exceeded file size) and there is no paste file in the QuickShop folder.
![No paste file](https://i.imgur.com/VYf54LD.png)

**Additional context**
The version of QuickShop I'm using is 4.0.6.0
![version](https://i.imgur.com/nFgCbLX.png)

**Comment by Ghost-chu** [comment]
please provide how you remove the shop and /qs paste link

**Comment by haloflooder** [comment]
We tried 2 methods to remove the chest and both methods reproduces the bug on our server. We tried `/qs remove` and just breaking the chest with an axe.

/qs paste doesn't work like I mentioned in the original post because of this ![image](https://i.imgur.com/toQLxRr.png)

But just to give more info that might be helpful:
server: 1.16.4 (Tuinity d1db107)
protocollib: 4.5.1
config: https://pastebin.com/gbvw5Bwr

**Comment by Ghost-chu** [comment]
Please update to 4.0.6.1 (QuickShop) and try again.
The config is invalid `This page is no longer available. It has either expired, been removed by its creator, or removed by one of the Pastebin staff.`
If still have the same issue, please upload another paste.

**Comment by Jackolans** [comment]
Putting the 'display-auto-despawn' in config to 'false' fixed the problem for me.

**Comment by bananasmoothii** [comment]
I have that but too, sometimes it works, sometimes it doesn't. Good thing is you can't pickup the items. Only way to remove the items is to destroy them, with lava. And I indeed have 'display-auto-despawn' on true.

**Comment by Ghost-chu** [comment]
Have tried VirtualDisplay? Does it only happening in legacy item mode?

**Comment by bananasmoothii** [comment]
For me it happens in Virtual Display mode

**Comment by mibby** [comment]
@Ghost-chu I'm also able to confirm this happening as of QuickShop-Remake-SNAPSHOT dev 564.

```
  display-items: true
  display-items-check-ticks: 6000
  display-type: 2
  display-auto-despawn: true
  display-despawn-range: 20
  display-check-time: 40
  display-allow-stacks: false
```

I spammed ``/qs create 1``. While it removed the existing shop to replace it with the new one, it did _not_ update the packet sent to the player to remove the old display item.
![link1](https://i.imgur.com/v15mxA0.png)

Relogging seemed to fix it. (Sign was on the other side)
![link2](https://i.imgur.com/MlsPS1k.png)

However when I removed the shop with ``/qs remove``, the floating items for every item the shop was created came back.
![link](https://i.imgur.com/jkwxt5B.png)

When I relogged immediately after, it did _not_ destroy the items from appearing. When I disconnected and waited some amount of seconds to relog, I relogged back with them not being present.
